### PR TITLE
Generate CSS file when make run and fix dupe variable declaration

### PR DESCRIPTION
**Pull request by charles1046**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,7 +41,7 @@ gen-css:
 	mkdir -p static/css/
 	sassc ./static/scss/main.scss ./static/css/main.css
 
-run: $(EXEC) html-updater
+run: $(EXEC) html-updater $(EXEC) gen-css
```
Comment (25077667) [inline Makefile:44]: The $(EXEC) is duplicated.

**Review by 25077667 — CHANGES_REQUESTED** [review]
Please remove the duplicated $(EXEC) tag.

**Comment by charles1046** [comment]
Updated

**Review by 25077667 — COMMENTED** [review]
Ok, thanks.

**Pull request merged**